tests/wpunit/AccessFunctionsTest.php

```diff
@@ -2232,15 +2232,21 @@ public function testRegisterConnectionWithUnderscores() {
 	}
 
 	public function testDeRegisterConnectionWithUnderscores() {
-
-		deregister_graphql_connection( 'test_field_with_underscores' );
+		deregister_graphql_connection( 'TestConnectionFieldWithUnderscores' );
```

Comment:
This change defeats the purpose of the test 🤔

Reply:
Intent of the test as I understood is that `deregister_graphql_connection()` also removes the connecting field even if the _field_ is in underscores.`deregister_graphql_field()` is tested separately for underscores.

Reply:
Or maybe it is a _type_ with underscores, and the `connectionTypeName` needs to be changed to include underscores?

Reply:
@justlevine I bet the intent was for the Type Name to have underscores. Because it is valid to have a Type like `My_Custom_Type` and I believe the intent of the test was to ensure that any normalization on the strings under the hood isn't stripping underscores, etc 🤷🏻‍♂️ 

Like I know the registry normalizes types to lowercase strings in some places, so this should be ensuring that a connection registered with underscores is also removed when deregistered